commands/channel_test.go

```diff
@@ -2176,3 +2176,169 @@ func (s *MmctlUnitTestSuite) TestRenameChannelCmd() {
 		s.Require().Equal(printer.GetLines()[0], updatedChannel)
 	})
 }
+
+func (s *MmctlUnitTestSuite) TestRemoveChannelUsersCmd() {
+	userEmail := "user@example.com"
+	userID := "example-user-id"
+	mockUser := model.User{Id: userID, Email: userEmail}
+	teamName := "teamName"
+	channelName := "channelName"
+	argsTeamChannel := teamName + ":" + channelName
+
+	s.Run("should remove user from channel", func() {
+		printer.Clean()
+
+		cmd := &cobra.Command{}
+		args := []string{argsTeamChannel, "user@example.com"}
+
+		foundTeam := &model.Team{
+			Id:          "teamId",
```

Comment:
These hardcoded values are duplicated all over the tests so we can move them to a constant and use those constants instead of the hardcoded values

Reply:
I've added some constants and replace them over all the tests (not only the ones for RemoveChannelUsersCmd). If you prefer me not to touch the other tests, let me know and I can revert those changes and only use the constants for the RemoveChannelUsersCmd use cases. Let me know!

Reply:
Great :), I prefer to have it that way. Let's wait for @mgdelacroix feedback in this case :)

Reply:
Agree with the approach :)